res/database/Default/database_programmes.xml

```diff
@@ -11514,10 +11512,9012 @@ Dream scenes and sentiments warp the storyline.</en>
+					<title>
+						<de>Jugendlicher Leichtsinn</de>
+						<en>Wild Youth</en>
+					</title>
+					<title_original>
+						<de>Wilde Jugend</de>
+						<en>Book I, Part 1</en>
+					</title_original>
+					<description>
+						<de>Orry Main verlässt seine Heimat in South Carolina für ein zweijähriges Offiziers-Training. Auf dem Weg dorthin trifft er die schöne Madeline Vabray und verliebt sich. In der Militär-Akademie freundet sich Orry mit dem Nordstaatler George Hazard an.</de>
+						<en>Orry Main leaves his home in South Carolina for two years of officer training. Along the way, he meets the beautiful Madeline Vabray and falls in love. At the military academy, Orry befriends northerner George Hazard.</en>
+					</description>
+					<ratings critics="78" speed="20" />
+				</programme>
+				<programme id="DF-Serien-DRAMA-Fackeln_im_Sturm_-_Staffel_01-E02" product="2" licence_type="2" tmdb_id="0" imdb_id="tt1338435" creator="8782" created_by="DerFronck">
+					<title>
+						<de>Diese Liebe ist falsch</de>
+						<en>Forbidden Love</en>
+					</title>
+					<title_original>
+						<de>Verbotene Liebe</de>
+						<en>Book I, Part 2</en>
+					</title_original>
+					<description>
+						<de>Orry kann nicht verwinden, dass Madeline den Plantagenbesitzer Justin geheiratet hat. Nur zu gerne zieht er mit seinem Freund George in den Krieg gegen Mexiko. An der Front sehen sie sich ihrem brutalen Ausbilder Bent aus der Akademie gegenüber.</de>
+						<en>Orry can't get over the fact that Madeline married Justin, a plantation owner. He is only too happy to go to war with his friend George against Mexico. On the front lines, they face their brutal instructor Bent from the academy.</en>
+					</description>
+					<ratings critics="76" speed="35" />
+				</programme>
+				<programme id="DF-Serien-DRAMA-Fackeln_im_Sturm_-_Staffel_01-E03" product="2" licence_type="2" tmdb_id="0" imdb_id="tt1338436" creator="8782" created_by="DerFronck">
+					<title>
+						<de>Die Diener der Oberen</de>
+						<en>Masters and Slaves</en>
+					</title>
+					<title_original>
+						<de>Herren und Sklaven</de>
+						<en>Book I, Part 3</en>
+					</title_original>
+					<description>
+						<de>Während George in Pennsylvania die Leitung der Eisenhütte übernimmt, bewirtschaftet Orry die Plantage seines verstorbenen Vaters. Die Sklaven-Frage droht das Land zu spalten und stellt auch Georges und Orrys Freundschaft auf eine harte Probe.</de>
+						<en>While George takes charge of the ironworks in Pennsylvania, Orry farms his deceased father's plantation. The slavery issue threatens to divide the country and also puts George and Orry's friendship to a severe test.</en>
+					</description>
+					<ratings critics="78" speed="23" />
+				</programme>
+				<programme id="DF-Serien-DRAMA-Fackeln_im_Sturm_-_Staffel_01-E04" product="2" licence_type="2" tmdb_id="0" imdb_id="tt1338437" creator="8782" created_by="DerFronck">
+					<title>
+						<de>Haßliebe</de>
```

Comment:
Hassliebe?

Reply:
fine with me

Reply:
(though the original title also uses old spelling.
I strongly suspect it's another thing that isn't consistently handled in the database, I *guess* there are some old titles that are updated to "neue Rechtschreibung", and some that aren't.

Reply:
I really only glanced through titles and descriptions. This was one that caught my eye.